test/unit/name_cleaner_test.py

```diff
@@ -0,0 +1,52 @@
+"""A dummy unit test so pytest has something to do."""
+import logging
+
+import pandas as pd
+
+from ferc_eia_match.name_cleaner import CompanyNameCleaner
+
+logger = logging.getLogger(__name__)
+
+
+def test_company_name_cleaner() -> None:
+    """Test the company name cleaner."""
+    dirty_df = pd.DataFrame(
+        {
+            "utility_name_dirty": [
+                "Duke Energy, LLC",
+                "  Duke Energy Limited Liability Company ",
+                "Duke Energy - [LLC]",
+                "Duke Energy l.l.c. (NC)",
+                "Pacific Gas & Electric Co.",
+                "Southern   Edison",
+                "Southern_Edison+",
+                r"{Southern} Edison:",
+            ]
+        }
+    )
+    expected_df = pd.DataFrame(
```

Comment:
Great!

I wonder if we should use this or something like it in our PUDL string normalizations. I guess when this eventually gets merged in we can think about it.